sessions/introduction.qmd

```diff
@@ -21,4 +21,127 @@ the keyboard.
+you as a team can focus on the content, on understanding what others are
+contributing to the project, and providing relevant feedback directly to
+where it is needed.
+
+So, to help with *knowing how exactly to work together*, here are the
+reasons why we use the tools and practices we do based on our
+experiences and research into what are the most optimal and effective
+ways to collaborate together as a team:
+
+-   We use Git and GitHub because they are both incredibly popular tools
+    for collaboration.
+-   We use the GitHub flow (using Git branches and pull requests)
+    because it allows each team member to work on their own part of the
+    project without interfering with others' work and makes it easy to
+    get feedback and reviews from others when that work is ready to be
+    added back to the main content of the project.
+-   We use "atomic" commits, which means the commit contains a single
+    conceptual change (e.g. only changes a typo), so that it is easier
+    for the reviewers (our team members) to understand what we changed
+    and why.
+-   We use the Conventional Commits standard to write commit messages
+    because it ensures that messages are consistent between team members
+    and it more clearly communicates what was done in a commit.
+-   We use GitHub pull requests to be able to directly comment on and
+    give feedback to changes made by other team members so that review,
+    discussions, and decisions can be visible to everyone on the team.
+
+For more reasons on why we use these tools and practices, see the
+[decision posts](https://decisions.seedcase-project.org) of our sibling
+the [Seedcase Project](https://seedcase-project.org).
+
+While the reasons for these choices are grounded in research,
+experience, and common practice in industry, there are costs related to
+these choices. The main one is that at this current time, few people who
+work in researcher use these practices. Part of the aim of this course
+is to spread awareness and education so that in the future more
+researchers and those in academia can be more familiar with the tools
+and practices that we will cover in this course.
+
+### How we will get there {.unnumbered}
+
+For this course, how you will learn to use more effective collaborative
+practices within a team setting is by:
+
+-   Participating in live-coding sessions where we will demonstrate how
+    to use Git and GitHub for collaboration.
+-   Discuss with your neighbour about what makes a collaborative
+    workflow effective.
+-   Working on hands-on exercises where you will practice using Git and
+    GitHub on making files, editing files, making pull requests,
+    reviewing pull requests, and merging pull requests.
+-   Work with at least one other person on a project, where you start
+    from the beginning and apply all the practices we've learned in the
+    course.
+
+### What it will look like {.unnumbered}
```

Comment:
I like having this section to show form the beginning what we're working towards 👍 🌟